libs/platform/src/lib/value-help-dialog/value-help-dialog/value-help-dialog.component.html

```diff
@@ -264,6 +264,7 @@ <h5 fd-title class="selected-and-condition-title" [id]="id + '-selected-item'">
 
 <ng-template #defineConditionsContent>
     <fdp-define-tab
+        class="fd-scrollbar"
```

Comment:
use directive instead of css class please.

Reply:
If you think it's not possible in this exact case, you have to import scrollbar's styles from fund-styles to the sass file of the component.